app-libs/stf/src/test_genesis.rs

```diff
@@ -67,9 +73,23 @@ pub fn test_genesis_setup(state: &mut SgxExternalities) {
 		(ALICE_ENCODED.into(), ALICE_FUNDS, ALICE_FUNDS),
 	];
 
+	fund_alice_evm_account(&mut endowees);
```

Comment:
With this name I would expect the function to modify an account that is passed in.
I think the function should either be named `append_funded_alice_evm_account` or it should return the account and the caller is responsible to add it to the list.

Reply:
+1 for returning the account as an output value